controllers/shops.js

```diff
@@ -0,0 +1,29 @@
+const { isEmpty } = require("lodash");
+const { shops } = require("../models");
+const moment = require("moment");
+
+module.exports = {
+  create: async (req, res) => {
+    try {
+      const { name, phoneNumber, address } = req.body;
+      const User = req.user;
+      if (isEmpty(name) || isEmpty(phoneNumber) || isEmpty(address)) {
+        throw { status: 400, message: "Required fields cannot be empty" };
+      }
+      const isShop = await shops.findOne({ where: { fk_user_id: User.id } });
```

Comment:
@Behramkhann , `isShop` is not a proper name. It's name should be simple `shop` or `shopFound`.
```suggestion
      let Shop = await shops.findOne({ where: { fk_user_id: User.id } });
```